memory/src/main/scala/filodb.memory/format/UnsafeUtils.scala

```diff
@@ -145,7 +143,7 @@ object UnsafeUtils {
     if (wordComp == 0) {
       var pointer1 = offset1 + minLenAligned
       var pointer2 = offset2 + minLenAligned
-      for { i <- minLenAligned until minLen optimized } {
+      for { i <- minLenAligned until minLen } {
```

Comment:
Missed a case here. Should use cforRange.

Reply:
Good catch!

Reply:
Went again through all the changes and didn't spot any more missed cases.